index.html

```diff
@@ -41,5 +41,149 @@ <h3>I’am a Software Developer</h3>
               <li class="social"><a href="#"><img src="img\medium.png"></a></li>
             </ul>
       </section>
+      
+      <!--Second section-MY WORK-->
+      <section class="middle">
+        <h1 class="headline">My recent works</h1>
+        <hr>
```

Comment:
- Kindly use the correct font style, image sizes, and spacing of elements on your project section to match the Figma design.
 ### Your Design

![image](https://user-images.githubusercontent.com/98453979/192097541-d122f3a6-c6f6-48bb-b71c-1f0944d3fbf0.png)

![image](https://user-images.githubusercontent.com/98453979/192097548-a9dd91ab-9984-4bb4-b761-97148ed9eeed.png)

### Figma Design

![image](https://user-images.githubusercontent.com/98453979/192097568-cb12687b-553e-409b-8a32-14a8075bd263.png)

![image](https://user-images.githubusercontent.com/98453979/192097573-4c52a4a7-7cd8-4016-be96-bff594d9d133.png)

Reply:
Hi @frank1738 
Thank you for your feedback 

I've pushed the changes, Kindly review them 

regards

Reply:
Good work @Zainab-Alzubair . Kindly submit your project on the dashboard for another review.

Reply:
Alright, thanks @frank1738